src/components/structures/MessagePanel.js

```diff
@@ -465,6 +465,12 @@ export default class MessagePanel extends React.Component {
                }
                return false;
             };
+            // events that we include in the group but then eject out and place
+            // above the group.
```

Comment:
Is it strange for it to be **above** the group instead of below? It's then displayed "before" the event "created the room" which seems like it should be first thing that happens... I guess it's okay if @nadonomy is okay with it, but it's a bit strange to read if actually look at all the pieces.

Reply:
yeah, iswym although I think the fact that it's pulled out as a separate tile makes it look more like a header than an out-of-order event. Another option would be to include the event in the summary so it's still in the right place if you expand the summary.

Reply:
Eh yeah, I guess I don't feel strongly enough any direction, so let's roll with this and tweak later.

Reply:
For posterity, @dbkr was on the money with it feeling more like a header. Putting the technology on a shelf— really it's just a notice, and users aren't concerned with what order the protocol requires things to happen in. Thanks to the way our brains work (https://en.wikipedia.org/wiki/Serial-position_effect) the best place for it is first.